ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -412,47 +412,93 @@ jobs:
       passed:
       {{ all_gating_jobs() | indent(6) }}
     - put: concourse-metadata-resource
-  - task: run_benchmarks
-    {{- alpine_tools_config()|indent(4) }}
-      params:
-        AWS_ACCESS_KEY_ID: ((benchmarks-access-key-id))
-        AWS_SECRET_ACCESS_KEY: ((benchmarks-secret-access-key))
-        AWS_DEFAULT_REGION: us-west-2
-        AWS_REGION: us-west-2
-        ARTIFACT_BUCKET: ((artifact-bucket))
-        BENCHMARKS_BRANCH: {{benchmarks.branch}}
-        BASELINE_BRANCH: {{benchmarks.baseline_branch}}
-        BASELINE_VERSION: {{benchmarks.baseline_version}}
-      run:
-        path: geode-ci/ci/scripts/run_benchmarks.sh
-      inputs:
-      - name: geode
-      - name: geode-ci
-      - name: geode-benchmarks
-      - name: concourse-metadata-resource
-      outputs:
-      - name: results
-    timeout: 4h
-    ensure:
-      do:
-      - task: cleanup_benchmarks
-        {{- alpine_tools_config()|indent(8) }}
-          params:
-            AWS_ACCESS_KEY_ID: ((benchmarks-access-key-id))
-            AWS_SECRET_ACCESS_KEY: ((benchmarks-secret-access-key))
-            AWS_DEFAULT_REGION: us-west-2
-            AWS_REGION: us-west-2
-            ARTIFACT_BUCKET: ((artifact-bucket))
-            BASELINE_BRANCH: {{benchmarks.baseline_branch}}
-            BASELINE_VERSION: {{benchmarks.baseline_version}}
-          run:
-            path: geode-ci/ci/scripts/cleanup_benchmarks.sh
-          inputs:
-          - name: geode
-          - name: geode-ci
-          - name: geode-benchmarks
-          - name: concourse-metadata-resource
-          - name: results
+  - aggregate:
```

Comment:
Given that 90% of the YAML below is duplicate, can we use a macro/template to produce it?

Reply:
Consider replace this with a loop
```
{% for ssl_var in ['', '-PwithSsl'] %}
- task: run_benchmarks{{ ssl_var.title }}
  < body of task >
 WITH_SSL: {{ ssl_var }}
  < remaining body of task >
{% endfor %}
```